packages/autorest.typescript/src/generators/static/licenseFileGenerator.ts

```diff
@@ -5,9 +5,9 @@ import { Project } from "ts-morph";
 import { getAutorestOptions } from "../../autorestSession";
 
 const mitLicenseText = `
-The MIT License (MIT)
+Copyright (c) Microsoft Corporation.
 
-Copyright (c) ${new Date().getFullYear()} Microsoft
```

Comment:
I have a minor question regarding License, @jeremymeng @xirzec do you think if it still makes sense to generate MIT license of Microsoft Corporation for unbranding scenarios?

Reply:
@qiaozha Great question! It probably should be up to customers to choose licenses for their generated code.  This is the same across languages. Better to confirm with codegen crew.